lib/DriverTool/autolink_extract_main.cpp

```diff
@@ -247,35 +247,31 @@ int autolink_extract_main(ArrayRef<const char *> Args, const char *Argv0,
   // in most object files
 
   std::vector<std::string> SwiftRuntimeLibsOrdered = {
```

Comment:
Would this benefit from a code comment for posterity that clarifies that ordering matters here and why it's ordered the way it is?

Reply:
There should probably also be a note that it's safe to list _all_ libraries in this list regardless of whether any given link operation actually uses any particular library or whether the library exists in the "current" toolchain (the list only controls ordering and deduplication).